### ISSUE TITLE
A getting started video on the Vendure YouTube channel

### ISSUE THREAD

**Issue by Raytle** [vendure-ecommerce/vendure#2619]
### Is your feature request related to a problem? Please describe.
Here's my problem: I want to easily understand how to set up and use Vendure.

### Describe the solution you'd like
I read this [guides/getting-started/installation/](https://docs.vendure.io/guides/getting-started/installation/). It's not bad; nonetheless, on [the official Vendure YouTube channel](https://www.youtube.com/@vendure_io) I would like to see a step-by-step video that demonstrates how to use Vendure after I have used Docker to install it in a container which assumes I am using SQLite.

### Describe alternatives you've considered
None.

### Additional context
I'm not a developer; therefore, I'd feel more comfortable watching a video on Youtube in which the presenter carefully walks through each step of the process of getting started with Vendure.

### Quick rant
Not everyone who wants to use Vendure is a developer. Really. They aren't. An excellent 15 to 30 minute "getting started video" on the Vendure YouTube channel would make it much easier for non-technical folks (like me) to grasp how to set up Vendure. SMH.

**Features are not the same things as benefits.** Most devs are addicted to adding feature after feature (see Gitlab, for example). Squashing bugs, and, gasp, writing documentation. Not so much.

But, well, see, here's the thing: if you fail to onboard customers in the first place, then, ummm, wellll, yeah... you are going to lose those potential customers. Repeat after me: potential customers are generally busy; therefore, you should enable them to easily grasp concepts that you already understand so well that you implicitly fail to grasp they need to be explained to potential customers.

If you think most potential non-technical folks will "just hop on Discord" to ask questions then I think you are deluding yourselves. Personally, I loathe Discord. I "just hoped on" another company's Discord a couple of times about five years ago to ask some questions about a software application. It was a big waste of time for me.

Finally, believe it or not, when  I see a link to Discord on a company's homepage, it normally raises a major red flag in my mind because I think, "This company is run by a bunch of ignorant techies who adore prattling on about their beloved technical product, but couldn't be bothered to properly document/explain it in videos. I hate dealing with these sorts of companies."

**Comment by michaelbromley** [comment]
Hi,

Thank you for your feedback. Vendure _is_ aimed at developers: you need to be able to minimally set up hosting, deploy a server, customize a storefront app, etc. I'd personally like to make it easier to get started - I spent hundreds of hours last year re-writing our documentation, and I have more ideas in the pipeline. But to make it accessible for non-developers is not a goal at the moment.

I'm interested to know: what's your reason as a non-developer to even check out Vendure rather than a truly no-code platform like Shopify?

**Comment by Raytle** [comment]
>  you need to be able to minimally set up hosting, deploy a server,

Not only is that assertion demonstrably false and remarkably ignorant, but, in your case, it is blatantly contradictory. 

For well over a decade, with little more than a few clicks of a mouse, it has been trivial to have a non-dev to spin up a server on a VPS offered at sites such as Hetzner, DigitalOcean, Vultr, Linode, etc.

> customize a storefront app, 

Do you mean modifying React or Angular? (Yeah, I read about Vendure v2.1)

> etc. 

I don't know what "etc." entails.


> I'm interested to know: what's your reason as a non-developer to even check out Vendure rather than a truly no-code platform like Shopify?

I'll let you, or rather one of your "writings", make one of my primary arguments for me...

[Why Open Source Is Winning
](https://www.vendure.io/blog/2023/09/why-open-source-is-winning
)Michael Bromley
Friday, September 8th 2023

In my case, this is one of my primary reasons for wanting a FOSS solution...

> Vendor lock-in
> 
> A coherent suite of tools from a single vendor can be a great way to efficiently solve a whole set of business problems. But what happens when something goes wrong? And by “goes wrong” we could be talking unexpected price hikes, gradual deterioration of the product, sudden shut-down of a critical part, or even the appearance of a much better alternative. All of these happen, and often.

> Extracting this vendor from your workflows can be more trouble that it is worth, so you may opt to suffer on with a host of tools that no longer serve you. That's lock in.

> OSS offers an alternative to getting locked in. Typically it is free, so price hikes don't apply. You aren't subject to sudden shut-downs since you own and fully control the solution. A popular project with a healthy community will see continual improvement.

**and, from that same technical guide**

> But running things yourself is only getting easier. With the rise of technologies like Docker and cloud platforms that increasingly focus on taking the pain out of operations, setting up an OSS replacement can be as simple as pointing your cloud host at a docker image and hitting “deploy”.

So, ummmm which is it? Are we talkin' 'bout late 90's early 2000's sys admins or are we talkin' 'bout modern DevOps? 'cause, no, I ain't down with the notion of caring for pets, but if I can simply herd some cattle, sure, why not?

Gee, a guy who opined, "setting up an OSS replacement can be as simple as pointing your cloud host at a docker image and hitting 'deploy'." blatantly contradicted himself when he falsely claimed, "you need to be able to minimally set up hosting, deploy a server..."

I've installed probably half a dozen shopping carts (all monoliths, none of this highfalutin headless stuff) over the last couple of decades. If I recall correctly they all ran on LAMP/LEMP stacks.

I'm not a dev, but I've got a pretty good aptitude for math/science, er, uh, I mean STEM sort of stuff. Also, these days I use ChatGPT to help me create Google Apps Scripts, Bash scripts, and even Python scripts to help me automate stuff for my business. I am particularly fond of PyAutoGUI.

If need be, I could go on Fiverr to find a dev to set Vendure up for me at on a VPS at Hetzner. In such a case I'd have him record a screencast with OBS (Open Broadcaster Software) so I'd be able to do it myself in the future.

Or, errrrr, ummmm ... you could put a screencast up on YouTube showing me how to do it.

What you seem to have failed to grasp is this: I suppose over 95% of **potential** users of Vendure in the USA would be very satisfied to start with a simple storefront which allowed them accept payments via Stripe and/or PayPal.

Call it "Vendure Lite", call it "Simple Vendure", call it "Vendure for Idiots". Call it whatever you like. But strip out the non-essential items, er, uh, I mean put them behind a feature flag, so that you can KISS (Keep It Simple Stupid). Give the people what they want: a 99 cent hamburger (well, maybe it's $2.49 these days) they can purchase from an app on their phone and pickup at the drive-thru window.

I urge you to take a month or two off from building more features for Vendure (engineers complain about clients pushing for scope creep, I complain about engineers addicted to adding more features)  so that you could "dumb things down", that is focus on usability by REMOVING features for "Vendure for Dummies", you could onboard myriad customers whom you have foolishly decided are outside of your current target market. 

Oh wait. Sorry. That wasn't harsh enough. Let me rephrase that... you could onboard myriad customers whom you have **ARROGANTLY AND IGNORANTLY** decided are outside of your current target market. I have a love hate relationship with engineers: I love to hate them!

"In low-end disruption, the disruptor is focused initially on serving the least profitable customer, who is happy with a good enough product. This type of customer is not willing to pay premium for enhancements in product functionality." [Low-end disruption](https://en.wikipedia.org/wiki/Disruptive_innovation#Low-end_disruption)

Clayton Christensen observation is a tenet what the "A" guy in a16z.com was talking about when he asserted that "Software is eating hardware."

**Comment by michaelbromley** [comment]
OK got it. You want me to make you a video tutorial. I'll add it to my to-do list 🤣

**Comment by Raytle** [comment]
Better yet, I suggest you ultimately pay someone on Fiverr, say, $50, to make a roughly thirty minute (half an hour) video for you. I suggest you contact React developer who lives in Eastern Europe or Latin America.

Initially, pay them $5 to create a 2 to 3 minute video. Why? That way you can test out a dozen or so people until you find someone you like.

Once you find someone you like, you can probably work with then in the future to create more videos for you.

Personally I like to use Google Doc because they enable easy collaboration. In other words, you can write up what you want in a Google Doc, then the Fiverr "seller" (on Fiverr, freelancers are called "sellers") can ask questions directly in the Google Doc. 

**Pro Tip:#1** Use the outline feature in Google Docs to make it easier to stay organized. (Get comfortable using "Tab" and "Shift-Tab" to promote or demote whole sections of an outline).

**Pro Tip:#2** Use Styles in Google Docs to format different portions of the outline with different size and different color fonts. It really helps. A lot.

**Pro Tip:#3** Instead of one big Google Doc, it's usually better to create multiple Google Docs which you hyperlink together (kind of, sort of like a wiki) so that you can easily archive portions that have been completed to get them out of the way by putting them in a folder called "Completed Items."
